src/sql/workbench/parts/notebook/notebook.component.ts

```diff
@@ -94,7 +95,8 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		@Inject(IViewletService) private viewletService: IViewletService,
 		@Inject(ICapabilitiesService) private capabilitiesService: ICapabilitiesService,
 		@Inject(ITextFileService) private textFileService: ITextFileService,
-		@Inject(ILogService) private readonly logService: ILogService
+		@Inject(ILogService) private readonly logService: ILogService,
+		@Inject(ITelemetryService) private _telemetryService: ITelemetryService
```

Comment:
>_telemetryService [](start = 37, length = 17)

nit: let's be consistent with _ vs no _ in the var names. The rest don't have a _ prefix